setup.py

```diff
@@ -15,4 +15,18 @@
     packages=find_packages("src"),
     package_dir={"": "src"},
     install_requires=[],
+    zip_safe=False,
+    include_package_data=True,
+    classifiers=[
+        "Development Status :: 1 - Planning",
+        "Intended Audience :: Science/Research",
```

Comment:
I think this is a relevant `Intended Audience` for the SDK, but I don't know if it applies in general to stdlib-utils. I don't think it's specifically required to have an `Intended Audience` classifier if it's not really applicable